docs/how-certificate-issuing-works.md

```diff
@@ -0,0 +1,70 @@
+# Certificate Issuing Overview
+
+This document walks through the process of issuing a code signing certificate
+from start to finish. This is a great entry point to understanding how Fulcio
+works if you're interesting in contributing to the project or just want to
+learn more about whats happening under the hood.
+
+## 1 | Certificate Request Input
+
+To begin, the client submits a certificate request to Fulcio.
+
+![Certificate request diagram](img/certificate-request.png)
+
+The certificate request contains three items:
+
+- An OIDC identity token. This is a JWT containing information about the
+  principal (identity of our client), the issuer (who verified this identity?
+Google, Github etc) and some other metadata.
+- The public key. This is the public half of an asymmetric key-pair generated
+  by the client. The public key will be embedded in the final x509 certificate
+if this request is successful
+- A challenge. This challenge proves the client is in possession of the private
+  key that corresponds to the public key provided. The challenge created by
+signing the subject portion of the OIDC ID token
```

Comment:
The challenge description reads correct, but is it stuffed into the cert now?

Reply:
I don't _think_ so. Are you saying it reads like that or it doesn't read like that and it should to be accurate?

Reply:
actually reading it again, I missed the reference to "request", so I misread it as being in the certificate 

> "The certificate request contains three items:"

Reply:
Ahhh right I see. Yeah I just meant the API request if that makes sense. I think I can clarify that